core/__tests__/modules/status.ts

```diff
@@ -178,4 +182,116 @@ describe("modules/status", () => {
+      });
+
+      it("shows sources that ran", async () => {
+        const source = await helper.factories.source();
+        await source.setOptions({ table: "users_table" });
+        await source.setMapping({ id: "userId" });
+        await source.update({ state: "ready" });
+
+        const schedule = await helper.factories.schedule(source);
+        const run = await helper.factories.run(schedule);
+
+        const _import = await helper.factories.import(
+          run,
+          undefined,
+          oldProfile.id
+        );
+        const now = new Date();
+        await _import.update({
+          profileAssociatedAt: now,
+          profileUpdatedAt: now,
+          groupsUpdatedAt: now,
+        });
+        await run.updateTotals();
+
+        const sources = await FinalSummaryReporters.Sources.getData();
+        expect(sources[0].name).toEqual(source.name);
+        expect(sources[0].importsCreated).toEqual(1);
+        expect(sources[0].profilesCreated).toEqual(0);
+        expect(sources[0].profilesImported).toEqual(1);
+        expect(sources[0].error).toEqual(null);
+      });
+    });
+    describe("it gathers destinations", () => {
+      it("does not show destinations without an export", async () => {
+        const destination = await helper.factories.destination();
+        const destinations = await Destination.findAll();
+        expect(destinations.length).toBe(1);
+        const destinationReport =
+          await FinalSummaryReporters.Destinations.getData();
+        expect(destinationReport).toEqual([]);
+      });
+
+      it("shows destinations that ran", async () => {
+        const destination = await helper.factories.destination();
+
+        const _export = await helper.factories.export(
+          undefined,
+          destination,
+          undefined
+        );
+        const _export2 = await helper.factories.export(
+          undefined,
+          destination,
+          undefined
+        );
+        const _export3 = await helper.factories.export(
+          undefined,
+          destination,
+          undefined
+        );
+        await changeTimestamps(_export3);
+
+        const now = new Date();
+        await _export.update({
+          completedAt: new Date(now),
+          state: "complete",
+        });
+
+        const destinations = await FinalSummaryReporters.Destinations.getData();
+        expect(destinations[0].name).toEqual(destination.name);
+        expect(destinations[0].exportsCreated).toEqual(2);
+        expect(destinations[0].exportsFailed).toEqual(0);
+        expect(destinations[0].exportsComplete).toEqual(1);
+      });
+    });
+  });
 });
+
+async function changeTimestamps(
+  instance: Profile | Export,
+  timestamp = "1900-01-01 12:13:14"
+) {
+  const tableName = instance.constructor["tableName"];
+  return api.sequelize.query(
+    `UPDATE ${tableName} SET "updatedAt" = '${timestamp}', "createdAt" = '${timestamp}' WHERE id = '${instance.id}'`
+  );
+}
```

Comment:
This function is because there is no good way with Sequelize to manually change `updatedAt`.  @tealjulia I think we have a method like this in a few places now.  Can you please move this into `@grouparoo/spec-helper`?   Then this method will be available via something like `helper.changeTimesamps(model)`